src/nvim/buffer.c

```diff
@@ -3942,8 +4059,9 @@ int build_stl_str_hl(
 
     // Otherwise if the item is a number, copy that to the output buffer.
     } else if (num >= 0) {
-      if (out_p + 20 > out_end_p)
-        break;                  /* not sufficient space */
+      if (out_p + 20 > out_end_p) {
+        break;                  //  not sufficient space
```

Comment:
Only one space should be after `//`, not two. I mention this for future reference. I will fix it in this PR.